@xen-orchestra/web/src/components/storage-repositories/panel/cards/StorageRepositoryHostsCard.vue

```diff
@@ -6,7 +6,7 @@
     </UiCardTitle>
     <UiCollapsibleList v-if="hosts.length > 0" tag="ul" :total-items="hosts.length">
       <li v-for="host in hosts" :key="host.id" v-tooltip class="text-ellipsis">
```

Comment:
```suggestion
      <li v-for="host in hosts" :key="host.id" v-tooltip>
```
I don't think ellipsis class is necessary yet, what do you think ?